### PR TITLE
{bio}[GCC/10.3.0,intel-compilers/2021.2.0] Flye v2.9 w/ Python 3.9.5

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
select-pika-c6gd-4xlarge-0001 - Linux centos linux 8.3.2011, AArch64, ARM UNKNOWN (graviton2), Python 3.6.8
See https://gist.github.com/232e8f09956422cdc48919bdb8de7d63 for a full test report.

File: easybuild/easyconfigs/f/Flye/Flye-2.9-GCC-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+source_urls = ['https://github.com/fenderglass/Flye/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['158ea620d4aa92a53dae1832b09fd605e17552e45b83eecbf28e41a4516a6957']
+
+dependencies = [('Python', '3.9.5')]
+
+download_dep_fail = True
+use_pip = True
+
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/f/Flye/Flye-2.9-GCC-10.3.0.eb:20]: Still needed, indeed, see https://github.com/easybuilders/easybuild-easyconfigs/pull/14223#issuecomment-955574328, thanks!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
select-pika-c6gd-4xlarge-0001 - Linux centos linux 8.3.2011, AArch64, ARM UNKNOWN (graviton2), Python 3.6.8
See https://gist.github.com/679322a27540775ef26f7702abae73d4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3117.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/d747bd38b812886f9a3af0575789bfb2 for a full test report.

File: easybuild/easyconfigs/f/Flye/Flye-2.9-GCC-10.3.0.eb (hunk 1 above)
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/f/Flye/Flye-2.9-GCC-10.3.0.eb:20]: @smoors fixed in 7cb0309

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
easybuild-c7.novalocal - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake, IBRS), Python 3.6.8
See https://gist.github.com/38b6d89feb9daf055def935901118ca9 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
jsfc01.int.jusuf.sebastian.cluster - Linux rocky linux 8.4, x86_64, AMD EPYC 7742 64-Core Processor (zen2), 1 x NVIDIA GRID V100-4C, 460.73.01, Python 3.6.8
See https://gist.github.com/7a94d25d078a6c809195ce36288af059 for a full test report.

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**